brambling/models.py

```diff
@@ -482,17 +551,44 @@ def get_absolute_url(self):
     def get_liability_waiver(self):
         return self.liability_waiver.format(event=self.name, organization=self.organization.name)
 
-    def editable_by(self, user):
-        return (
-            self.organization.editable_by(user) or (
-                user.is_authenticated() and
-                user.is_active and
-                self.additional_editors.filter(pk=user.pk).exists()
+    def get_permissions(self, person):
```

Comment:
We could make this function more wieldy (maybe) by adding a bunch of permissions checkboxes to the models... but that would add its own complications. I think this is as good as we can do for now? Definitely way better than the old version.